### PR TITLE
kurara-nikgapps-sl

### PR DESCRIPTION
**Pull request by nycctoo**
i hope this gapps working fine thanks

### CODE REVIEW

**Comment by nikgapps** [comment]
We cannot merge the pull request due to following reasons

- 11/nyccto-gapps.config is not on the latest version of NikGapps Config, please update the config file to version 23

Kindly make the changes and send the pull request again!

**Comment by nycctoo** [comment]
im using version 23 sir

Pada tanggal Sen, 24 Okt 2022 16.54, Nikhil Menghani <
***@***.***> menulis:

**Comment by nikgapps** [comment]
We cannot merge the pull request due to following reasons

- 11/nyccto-gapps.config is not on the latest version of NikGapps Config, please update the config file to version 23

Kindly make the changes and send the pull request again!

File: 11/nyccto-gapps.config [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# NikGapps configuration file
+
+# If you are not sure about the config, just skip making changes to it or comment it by adding # before it
+# visit https://nikgapps.com/misc/2020/11/22/NikGapps-Config.html to read everything about nikgapps
+
+Version=22
```
Comment (nikhilmenghani) [inline 11/nyccto-gapps.config:6]: Version is 22, should be latest i.e. 23

**Comment by nikgapps** [comment]
Closing as no action was performed in last 24 hours

**Pull request closed without merging**